### DISCUSSION TITLE
deployment error

### DISCUSSION THREAD

**Discussion by akanz** [vercel/next.js#35205] [Help]
So I am kinda getting this strange error when deploying on vercel. What should I do about it? from the error, it appears it's from vercel.
<img width="1440" alt="Screenshot 2022-03-10 at 12 47 09" src="https://user-images.githubusercontent.com/33222698/157656143-20d5aad8-74c8-4254-baa5-c288542319c4.png">

**Comment by icyJoseph** [comment]
Is this still happening? Might've been temporary service degradation?

**Reply by akanz** [reply]
yes it is. I even deleted the project and the team entirely and deployed from scratch, still the same error

**Reply by icyJoseph** [reply]
Alright, is there any code we could take a look at, or any possibility to describe what is it that you are deploying?

**Reply by akanz** [reply]
okay, it's an organization repo so sharing might not be an option. It is a basic Next app with Serverside props. the very first time I deployed, it was successful but I tried adding a custom domain and I probably messed it up. deleted the project and redeployed and it's been like that.
